x/genutil/client/cli/migrate.go

```diff
@@ -6,24 +6,24 @@ import (
 	"sort"
 	"time"
 
-	"github.com/cosmos/cosmos-sdk/types/errors"
-
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/version"
 	extypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
-	"github.com/spf13/cobra"
-	tm "github.com/tendermint/tendermint/types"
-
 	v0130 "github.com/desmos-labs/desmos/x/genutil/legacy/v0.13.0"
+	v0150 "github.com/desmos-labs/desmos/x/genutil/legacy/v0.15.0"
 	"github.com/desmos-labs/desmos/x/genutil/types"
+	"github.com/spf13/cobra"
+	tm "github.com/tendermint/tendermint/types"
 )
 
 // migrationMap contains the list of migrations that should be performed when migrating
 // a version of the chain to the next one. It contains an array as we need to support Cosmos SDK migrations
 // too if needed.
 var migrationMap = map[string]types.MigrationCallback{
 	"v0.13.0": v0130.Migrate,
+	"v0.15.0": v0150.Migrate,
```

Comment:
I'd prefer we call this `stargate` as  I would like to call version 0.15.0 this way. No need to change the packages names though